### DISCUSSION TITLE
Multi Camera board V2 Issues

### DISCUSSION THREAD

**Discussion by Sohaib448506** [robotology/community#488] [Q&A] [answered]
Hi, world
I am trying to connect two camera's with Raspberry Pi 4. To connect two camera's, i am using multi-camera board V2. But it is giving the firmware error when i am trying to capture pictures. If a attach the single camera directly to the raspberry Pi4, it take pictures and there is no error. The connections are properly done. But again this error is coming. Kindly help me out to find solution. 1. Does the firmware version effect to use raspberry pi 4 with multi board v2?
2. Since, it is 4 channel camera module. How to use and configure the multi-camera board v2 for single or two cameras connected to it.
Thanks

**Comment by pattacini** [comment, marked as answer]
Hi @Sohaib448506 

I'm afraid that your question is off-topic here as our community covers the iCub and the YARP ecosystems.
You might consider posting on https://robotics.stackexchange.com instead.

Best

**Reply by Sohaib448506** [reply]
Ok @pattacini ..Thanks
